packages/node_modules/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -248,6 +249,36 @@ class Transforms {
             }));
         }
 
+        // Decrypt meeting title if present
+        if (activity?.meeting?.title) {
```

Comment:
I've wanted something like Optional Chaining for a long time but could not find it. Thanks for the tip @mwegman.